SIAB/io/read_input.py

```
@@ -155,7 +155,15 @@ def is_reference_systems_line(key, value):
                         if isinstance(value[2], list):
                             return True
         return False
-    
+
+    #               STRU1      dimer     8       2       1   1.8  2.0  2.3  2.8  3.8
+    refsys = r'^\s*(STRU\d+)\s+(\w+)\s+(\d+)\s+(\d+)\s+(\d+)\s+((\d+\.\d+\s+)+)$'
+    #             Level1      STRU1       4      none    1s1p   
+    orb = r'^\s*(Level\d+)\s+(STRU\d+)\s+(\d+)\s+(\w+)\s+([a-z0-9]+)$'
+    #         Save2    Level2      DZP
+    save = r'^\s*(Save\d+)\s+(Level\d+)\s+([A-Z0-9]+)$'
```

Ruff

Local variable `save` is assigned to but never used

Remove assignment to unused variable `save`

(F841)